src/relay/pass/pattern_util.h

```diff
@@ -415,6 +415,13 @@ static inline Expr Full(Expr fill_value,
   return CallNode::make(op, {fill_value}, Attrs(attrs), {});
 }
 
+static inline Expr Concatenate(Expr data, int axis) {
```

Comment:
this is actually the same as `MakeConcatenate` below isn't it?

Reply:
Yup. Nice observation. Made the changes to use that.